sp800-63c/sec8_security.md

```diff
@@ -5,196 +5,60 @@
-
--   *Assertion redirect* - To mitigate this threat, the assertion
-    includes the identity of the RP for which it was generated. The RP
-    verifies that incoming assertions include its identity as the
-    recipient of the assertion. 
-
--   *Assertion reuse* - To mitigate this threat, the following
-    mechanisms are used:
-
-	1.  The assertion includes a timestamp and has a short lifetime
-    of validity. The RP checks the timestamp and lifetime values to
-    ensure that the assertion is currently valid. 
-	2.  The RP keeps track of assertions that were consumed within
-    a configurable time window to ensure that an assertion is not
-    used more than once within that time window. 
-
--   *Secondary authenticator manufacture* - To mitigate this threat, one
-    of the following mechanisms is used:
-
-	1.  The secondary authenticator may contain sufficient entropy that an
-    attacker without direct access to the IdP's random number
-    generator cannot guess the value of a valid secondary authenticator.
-
-	2.  The secondary authenticator may contain timely assertion data that
-    is signed by the IdP or integrity protected using a key shared
-    between the IdP and the RP.
-
-
-
--   *Secondary authenticator capture* - To mitigate this threat,
-    adequate protections are in place throughout the lifetime of
-    any secondary authenticators used in the assertion protocol.
-
-	1.  In order to protect the secondary authenticator while it is in
-    transit between the IdP and the subscriber, the secondary
-    authenticator is sent via an authenticated protected channel established
-    during the primary authentication of the subscriber. 
-
-	2.  In order to protect the secondary authenticator from capture as it
-    is submitted to the RP, the secondary authenticator is used in
-    an authentication protocol which protects against eavesdropping and
-    man-in-the-middle attacks.
-
-	3.  In order to protect the secondary authenticator after it has been
-    used, it is never transmitted over an unprotected channel or to
-    an unauthenticated party while it is still valid.
-    
--   *Assertion substitution* - To mitigate this threat, one of the
-    following mechanisms is used:
-
-	1.  Responses to assertion requests contain the value of the assertion reference used
-    in the request or some other nonce that was cryptographically bound
-    to the request by the RP.
-
-	2.  Responses to assertion requests are bound to the corresponding
-    requests by message order, as in HTTP, provided that assertions and
-    requests are protected by a protocol such as TLS that can detect and
-    disallow malicious reordering of packets.
+Because of the additional party, the IdP, in the authentication process, there are additional opportunities for attackers to compromise federated identity transactions. This section summarizes many of the attacks and their mitigations that are applicable when federation is being used.
```

Comment:
Wording is awkward here and not exactly accurate since the IdP isn't an "additional party" so much as an additional role. Suggest:

> Since the federated authentication process involves coordination between multiple components, including the CSP which now acts as an IdP, there are additional opportunities for attackers to compromise federated identity transactions. This section...